plugins/woocommerce/templates/single-product/product-attributes.php

```diff
@@ -23,10 +23,10 @@
 	return;
 }
 ?>
-<table class="woocommerce-product-attributes shop_attributes">
+<table class="woocommerce-product-attributes shop_attributes" aria-label="<?php echo esc_html_e( 'Product Details', 'woocommerce' ); ?>">
```

Comment:
Think this got missed (which is my fault for adding two separate comments on the same line):

```suggestion
<table class="woocommerce-product-attributes shop_attributes" aria-label="<?php esc_attr_e( 'Product Details', 'woocommerce' ); ?>">
```